Sources/tart/OCI/Registry.swift

```diff
@@ -214,8 +214,7 @@ class Registry {
         parameters: lastChunk ? ["digest": digest] : [:],
         body: chunk
       )
-      let expectedStatus = lastChunk ? HTTPCode.Created.rawValue : HTTPCode.Accepted.rawValue
-      if response.statusCode != expectedStatus {
+      if response.statusCode != HTTPCode.Created.rawValue && response.statusCode != HTTPCode.Accepted.rawValue {
```

Comment:
Do you think it would be good idea to leave a comment here since it's working according to a specification anymore?